src/Sulu/Bundle/MediaBundle/FileInspector/UploadFileSubscriber.php

```diff
@@ -51,21 +51,34 @@ public function onKernelRequest(RequestEvent $event): void
     {
         $request = $event->getRequest();
 
-        /**
-         * @var string $key
-         * @var UploadedFile $file
-         */
-        foreach ($request->files as $key => $file) {
+        $request->files->replace($this->inspectFiles($request->files->all()));
+    }
+
+    /**
+     * @param array<string, mixed> $files
+     *
+     * @return array<string, mixed>
+     */
+    private function inspectFiles(array $files): array
+    {
+        foreach ($files as $key => $file) {
+            if (\is_array($file)) {
+                $files[$key] = $this->inspectFiles($file);
+                continue;
+            }
+
+            $mimeType = $file->getClientMimeType();
```

Comment:
```suggestion
            $mimeType = $file?->getClientMimeType();
```

We're using this in production and it crashes with "Uncaught Error: Call to a member function getClientMimeType() on null"

Reply:
Yes, the original line will surely crash if $file is null, so your change makes sense.

Reply:
@mamazu @mhnrm done :)